python/lsst/afw/table/base.py

```diff
@@ -0,0 +1,219 @@
+from __future__ import absolute_import, division, print_function
+
+from past.builtins import basestring
+
+from .record import addRecordMethods
+from .table import addTableMethods
+from .catalog import addCatalogMethods
+from ._base import BaseRecord, BaseTable, BaseCatalog
+
+import numpy as np
+
+__all__ = []  # import this module only for its side effects
+
+
+def addBaseRecordMethods(cls):
+    """Add pure Python methods to BaseRecord
+    """
+    addRecordMethods(cls)
+
+    def _set(self, key, value):
+        """
+        Given a string or `lsst.afw.table.Key`, find the `BaseRecord` for the given key and set its value.
```

Comment:
Docstring here (and below) is incorrect: we're not "finding" a `BaseRecord`; this is a a method on `BaseRecord`, and we're just setting the value of a field with it.